Sources/Plasma/FeatureLib/pfMetalPipeline/plMetalDevice.h

```diff
@@ -190,6 +190,15 @@ class plMetalDevice
     MTL::PixelFormat GetFramebufferFormat() const { return fFramebufferFormat; };
     
     MTL::Library* GetShaderLibrary() const { return fShaderLibrary; }
+    
+    static constexpr MTL::StorageMode GetDefaultStorageMode()
+    {
+#if TARGET_OS_MAC
```

Comment:
For consistency with the rest of the engine, we might want to use `HS_BUILD_FOR_MACOS` here (but we should double check that's actually only getting set on macOS through CMake).

IIRC we have two defines that have mostly been conflated thus far:
* `HS_BUILD_FOR_APPLE`
* `HS_BUILD_FOR_MACOS`

Reply:
I'll change to 'HS_BUILD_FOR_MACOS'. We don't have an 'HS_BUILD_FOR_IOS' but we also don't require that here.

Reply:
Nobody has been... courageous... enough to make `HS_BUILD_FOR_IOS` exist yet 😂